cfme/tests/intelligence/reports/test_reports_schedules.py

```diff
@@ -179,3 +180,24 @@ def test_reports_disable_enable_schedule(appliance, schedule):
     assert not schedule.enabled
     schedules.enable_schedules(schedule)
     assert schedule.enabled
+
+
+@pytest.mark.ignore_stream("5.10")
+def test_reports_disable_enable_schedule_from_summary(appliance, schedule):
+    """
+    This test checks if schedule can be enabled/disabled from it's summary page.
+
+    Polarion:
+        assignee: pvala
+        casecomponent: Reporting
+        initialEstimate: 1/10h
+    Bugzilla:
```

Comment:
**Optional** empty line between the `Polarion` block and the `Bugzilla` block. I like it for readability.